Cargo.toml

```
@@ -9,7 +9,7 @@ lazy_static = "1"
 serde = "1.0"
 serde_json = "1.0"
 wasmer = "2.2.1"
-as-ffi-bindings = { "git" = "https://github.com/massalabs/as-ffi-bindings.git", branch = "fix_memory_errors" }
+as-ffi-bindings = { "git" = "https://github.com/massalabs/as-ffi-bindings.git" }
```

You can use the latest version `as-ffi-bindings = "0.2.3-anyptr"`
```suggestion
as-ffi-bindings = "0.2.3-anyptr"
```